include/sacn/cpp/source.h

```diff
@@ -133,22 +134,26 @@ class Source
     UniverseSettings(uint16_t universe_id, const uint8_t* new_values_buffer, size_t new_values_size);
 
     bool IsValid() const;
-  };
 
+  private:
+    std::vector<EtcPalIpAddr> c_unicast_destinations;
```

Comment:
I'm not understanding why this is needed. Internally, the settings would be translated & used, right?  Not re-stored here..

Reply:
I thought that the source API would need to keep the data, but then I realized it will just copy the unicast dests into its internal state that is shared with the add/remove dest functions. Good catch, thanks!

Reply:
Wait... the memory still needs somewhere to live just to get passed to the function. Let's talk.

Reply:
@ChristianReese like I said, take a look at how it's done in RDMnet: https://github.com/ETCLabs/RDMnet/blob/develop/include/rdmnet/cpp/device.h#L479

Reply:
Going with that.